### DISCUSSION TITLE
GitHub Issue Template: Routing Issues: Dead Link

### DISCUSSION THREAD

**Discussion by Zian** [osmandapp/OsmAnd#14317] [Q&A]
Where should I report a dead link in the text that is displayed when reporting a new routing-related issue?

The text says "`Please be sure to read our [FAQ](https://osmand.net/help-online) before creating an issue here.`"

After I visit the URL, I see:

> 403 Forbidden
> nginx/1.14.0 (Ubuntu)

**Comment by vshcherb** [comment]
Fixed https://github.com/osmandapp/OsmAnd/commit/1d5153d561616d78023722dd2ec76f44c2187224
